django_cloud_tasks/base.py

```diff
@@ -161,27 +171,31 @@ def __init__(self, body):
         self.setup()
 
     def setup(self):
-        payload = self.body['task']['appEngineHttpRequest']['body']
+        payload = self.body["task"]["appEngineHttpRequest"]["body"]
```

Comment:
this is appengine specific, I think our task `payload` has to look different.

Reply:
fixed